src/style.css

```diff
@@ -0,0 +1,143 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+.page {
+  margin: 0;
+  padding: 0;
+
+  --text-color: #060b35;
+  --sub-text-color: #616070;
+  --button-color: #00acdc;
+  --border-card-color: #f3f3f3;
+  --white-color: #fff;
+}
+
+.card {
+  width: 166px;
+  border: 1px solid var(--border-card-color);
+  background-color: var(--white-color);
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  font-family: "Roboto", sans-serif;
+}
+
+/* #region image */
+.card__image {
+  width: 160px;
+  height: 134px;
+}
+/* #endregion */
+
+/* #region title/subtitle */
+.card__title {
+  margin: 40px 0 0;
+  padding: 0;
+
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--text-color);
+}
+
+.card__subtitle {
+  margin-top: 4px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--sub-text-color);
+  align-self: start;
+}
+/* #endregion */
+
+/* #region Reviews */
+.card__reviews {
+  margin-top: 16px;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+}
+
+.card__reviews-number {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  align-self: center;
+}
+/* #endregion */
+
+/* #region stars  */
+.stars {
+  display: flex;
+}
+
+.star_star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--1 > :nth-child(-n+1),
+.stars--2 > :nth-child(-n+2),
+.stars--3 > :nth-child(-n+3),
+.stars--4 > :nth-child(-n+4),
+.stars--5 > :nth-child(-n+5) {
+  background-image: url(images/star-active.svg);
+}
+/* #endregion */
+
+/* #region price */
+.card__price-area {
+  margin-top: 24px;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+}
+
+.card__price-text {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--sub-text-color);
+}
+
+.card__price-number {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--text-color);
+}
+/* #endregion */
+
+/* #region btn-link */
+.card__button {
+  margin-top: 16px;
+  width: 164px;
+  height: 38px;
```

Comment:
its better not hardcode height for elements with content inside. Use paddings instead